apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs

```diff
@@ -81,35 +81,33 @@ public void ReadFile(IExceptionLogger exceptionLogger)
         // End sample
 
         // Sample: RegisterGoogleLogger
-        public void Configure(ILoggerFactory loggerFactory)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
         {
             string projectId = "[Google Cloud Platform project ID]";
-            loggerFactory.AddGoogle(projectId);
+            loggerFactory.AddGoogle(app.ApplicationServices, projectId);
         }
         // End sample
 
-#if NETCOREAPP2_0
         private class Startup { }
-        public void Configure()
+        public static void Program_Main()
         {
-            // Sample: RegisterGoogleLogger2
+            // Sample: UseGoogleDiagnostics
```

Comment:
Adding a new sample will also require updated our [index.md](https://github.com/GoogleCloudPlatform/google-cloud-dotnet/blob/master/apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md) file

Reply:
How can  I do this?

Reply:
Here is an [example](https://github.com/GoogleCloudPlatform/google-cloud-dotnet/commit/b107f866fc148c036b4aaca50b0b939e0eb727a8#diff-92526e1df9484aa96d1d78717da6d068) of updating the file.  It's not the most straight froward of how this all works so feel free to ping me if you need a hand here.